typemap.py

```diff
@@ -0,0 +1,61 @@
+from decimal import Decimal
```

Comment:
Are you going to turn this into a set of unittests in test_bson.py?

Reply:
Just did. See `test_bson_custom_types.py`.